std/stream.d

```diff
@@ -3064,7 +3064,6 @@ class MmFileStream : TArrayStream!(MmFile) {
     if (isopen)
     {
       super.close();
-      delete buf;
```

Comment:
> Please leave std.stream alone. The only fixes to be made on it are those that allow it to continue to compiler and work. It's deprecated.

The idea of @JackStouffer is to forbid the use of `delete` within the entire codebase with DScanner and at the moment DScanner doesn't allow to exclude specific blocks, hence this modification.